_includes/nav_v1.0.html

```diff
@@ -5,8 +5,8 @@
 	<li class="navContainer">
 		<a href="{{ site.url }}/v1.0/docs/whatisk8s.html" class="nav-link">What is Kubernetes?</a>
 	</li><!-- 
-					GETTING STARTED
-					 -->
+GETTING STARTED
```

Comment:
Can you bring it up on your own gh-pages so I can see it?

On Fri, Jul 31, 2015 at 11:14 AM, RichieEscarez <notifications@github.com>
wrote:

> In _includes/nav_v1.0.html
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/12092#discussion_r36000057>
> :
>
> > @@ -5,8 +5,8 @@
> >  	<li class="navContainer">
> >  		<a href="{{ site.url }}/v1.0/docs/whatisk8s.html" class="nav-link">What is Kubernetes?</a>
> >  	</li><!--
> > -					GETTING STARTED
> > -					 -->
> > +GETTING STARTED
>
> Just for easier scanning (since the page is so long)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/12092/files#r36000057>
> .
>